packages/fx-core/resource/package.nls.json

```diff
@@ -529,6 +529,33 @@
   "core.copilot.addAPI.success": "%s have(has) been successfully added to %s",
   "core.copilot.addAPI.InjectAPIKeyActionFailed": "Inject API key action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.",
   "core.copilot.addAPI.InjectOAuthActionFailed": "Inject OAuth action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.",
+  "core.uninstall.botNotFound": "Cannot find bot using the manifest ID %s",
+  "core.uninstall.confirm.tdp": "App registration of manifest ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.m365App": "Microsoft 365 Application of Title ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.bot": "Bot framework registration of bot ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.cancel.tdp": "Uninstallation of app registration is cancelled.",
+  "core.uninstall.confirm.cancel.m365App": "Uninstallation of Microsoft 365 Application is cancelled.",
+  "core.uninstall.confirm.cancel.bot": "Uninstallation of Bot framework registration is cancelled.",
+  "core.uninstall.success.tdp": "App registration of manifest ID: %s has been successfully uninstalled.",
+  "core.uninstall.success.m365App": "Microsoft 365 Application of Title ID: %s has been successfully uninstalled.",
+  "core.uninstall.success.delayWarning": "The uninstallation of the Microsoft 365 Application may be delayed.",
+  "core.uninstall.success.bot": "Bot framework registration of bot ID: %s has been successfully uninstalled.",
+  "core.uninstall.failed.titleId": "Unable to find the Title ID. This app is probably not installed.",
+  "core.uninstallQuestion.manifestId": "Manifest ID",
+  "core.uninstallQuestion.env": "Environment",
+  "core.uninstallQuestion.titleId": "Title ID",
+  "core.uninstallQuestion.chooseMode": "Choose a way to clean up resources",
+  "core.uninstallQuestion.manifestIdMode": "Manifest ID",
+  "core.uninstallQuestion.manifestIdMode.detail": "Clean up resources associated with Manifest ID. Resources including application registration in Teams Developer Portal, bot registration in Bot Framework Portal and uploaded custom applications in Microsoft 365 apps. The Manifest ID can be found in the environment file (with the default environment key Teams_App_ID) in the project generated by the Teams Toolkit.",
```

Comment:
Proposed: Clean up resources associated with Manifest ID. This includes app registration in Teams Developer Portal, bot registration in Bot Framework Portal, and custom apps uploaded to Microsoft 365. You can find the Manifest ID in the environment file (default environment key: Teams_App_ID) in the project created by Teams Toolkit.